examples/live/check_results.py

```diff
@@ -1,208 +1,236 @@
-            single_fail = True
+                if not test_all.all():
+                    log.error('Invalid template parameters in %s %s',
+                              trigfp, detector)
+                    single_fail = True
+
+    # check that triggers were produced in all detectors
+    if detectors_with_trigs != tested_detectors:
+        missing = sorted(tested_detectors - detectors_with_trigs)
+        log.error('No triggers found in %s', ', '.join(missing))
+        single_fail = True
       
-      if single_fail:
-            log.error('Single Trigger Test Failed')
-      return single_fail
+    if single_fail:
+        log.error('Single Trigger Test Failed')
+    return single_fail
+
 
 def check_coinc_results(inj_table):
-      coinc_fail = False
-      print_template_parameters = False
-      # gather coincident triggers
-      coinc_trig_paths = sorted(glob.glob('output/coinc*.xml.gz'))
-      l=len(coinc_trig_paths)
-      if l==0:
+    coinc_fail = False    
+    # gather coincident triggers
+    coinc_trig_paths = sorted(glob.glob('output/coinc*.xml.gz'))
+    n_coincs=len(coinc_trig_paths)
+    if n_coincs==0:
+        if verbose:
             log.error('No coincident triggers detected')
-            coinc_fail = True
-      elif l>=10: 
+        coinc_fail = True
+    elif n_coincs>=10: 
+        if verbose:
             log.error('Too many coincident triggers detected')
-            coinc_fail = True
-      else: 
-            log.info('%d coincident trigger(s) detected', l)
+        coinc_fail = True
+    elif verbose: 
+        log.info('%d coincident trigger(s) detected', n_coincs)
       
-      #get properties of injection
-      inj_mass1=inj_table.get_column('mass1')[0]
-      inj_mass2=inj_table.get_column('mass2')[0]    
-      inj_spin1z=inj_table.get_column('spin1z')[0] 
-      inj_spin2z=inj_table.get_column('spin2z')[0] 
-      inj_time=inj_table.get_column('end_time')[0]
-      inj_snr=15
+    with h5py.File('test_inj.hdf', 'r') as inj:
+        inj_mass1 = inj['mass1'][:]
+        inj_mass2 = inj['mass2'][:]
+        inj_spin1z = inj['spin1z'][:]
+        inj_spin2z = inj['spin2z'][:]
+        inj_time = inj['tc'][:]
+         
+    n_injs = len(inj_mass1)
+    if n_injs > n_coincs :
+        log.error('More injections than coincident triggers')
+        coinc_fail = True
+    if n_injs < n_coincs :
+        log.error('Fewer injections than coincident triggers')
+          
       
-      #print properties of coincident triggers
-      for ctrigfp in coinc_trig_paths:            
-            xmldoc = ligolw_utils.load_filename(
+    #create field array to store properties of triggers
+    trig_props = record.FieldArray(n_coincs, dtype=[('mass1', float), \
+                ('mass2', float), ('spin1z', float), ('spin2z', float), \
+                ('tc', float), ('net_snr', float)])
+      
+    #store properties of coincident triggers
+    x=-1
+    for ctrigfp in coinc_trig_paths:     
+        x+=1
```

Comment:
If you are using `x` as an index for each coinc, there is a neat one-line way to achieve this:
```python
for x, ctrigfp in enumerate(coinc_trig_paths):
```